### PR TITLE
Init back

### LINKED ISSUE #1: Application boot

### PR DESCRIPTION
**Pull request by als-v**
Closes #1

### CODE REVIEW

**Review by JuanFSR — COMMENTED** [review]
Good job!

**Review by caiotheodoro — COMMENTED** [review]
dale

File: package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "dependencies": {
+    "commitizen": "^4.2.4",
+    "express": "^4.17.3",
+    "git-cz": "^4.8.0",
+    "pg": "^8.7.3",
+    "reflect-metadata": "^0.1.13",
+    "typeorm": "^0.2.45",
+    "typescript": "^4.6.2",
+    "yarn": "^1.22.17"
```
Comment (dennisurtubia) [inline package.json]: Removes `yarn` dependency. We can use `npm`.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,119 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+lerna-debug.log*
+
+# lock
+*.lock
```
Comment (dennisurtubia) [inline .gitignore]: We need to version the `lock` file.

**Review by dennisurtubia — CHANGES_REQUESTED** [review]

**Pull request merged**